test/passes/remove-unused-brs_all-features.txt

```diff
@@ -505,10 +525,11 @@
   )
   (loop $in-todo
    (if
-    (i32.const 0)
-    (block $out-todo
+    (i32.eqz
+     (i32.const 0)
     )
```

Comment:
why did these changes happen?  i'd guess because enabling all features means the call can now branch due to exceptions. in that case, how about just enabling multivalue? or create a new file for the new test and enable all features there. i lean towards the latter.

Reply:
I lean toward the former because I think the single extra test does not warrant a separate file.